README.md

```diff
@@ -2,6 +2,8 @@ sass-maven-plugin
 =================
 # NOTICE: Plug-in no longer supported
 
+[![Maintenance](https://img.shields.io/maintenance/no/2015.svg)]()
```

Comment:
:art: wrapping link around image is not needed
Maybe adjust alt text to "not maintained"?